src/Pages/Home/components/GnatAndPMButtonMenu.tsx

```diff
@@ -2,10 +2,14 @@ import { useState } from "react"
 import { DoubleArrowDown } from "../../../components/Icons/DoubleArrowDown"
 import { OpenArrowsSvg } from "../../../components/Icons/OpenArrowsSvg"
 import { GnatDetails } from "./GnatDetails"
+import { AddOperationalCommand } from "./AddOperationalCommand"
 
 export function GnatAndPMButtonMenu() {
     const [isGnatOpen, setIsGnatOpen] = useState(false)
+    const [isPMOpen, setIsPMOpen] = useState(false)
     return <div>
+      {isPMOpen && <AddOperationalCommand />}
```

Comment:
this return its not write correctly:
we need to write it more cleaner and structured 
like this:

```
return (
    <div>
        {isPMOpen && (
            <>
            <AddOperationalCommand `/>`
                <span>new PM</span>
            </>
        )}
        <div className="flex justify-between">
            <button
                className="p-2 bg-primary text-primary-foreground flex gap-2"
                onClick={() => setIsGnatOpen(!isGnatOpen)}
            >
                <DoubleArrowDown
                    className={`transition-all ${isGnatOpen ? 'rotate-180' : ''}`}
                />
            </button>
           ......
        </div>
    </div>
);
```